src/cms-content/footer/footer.json

```diff
@@ -1,19 +1,6 @@
 {
   "aboutUs": "The Act Now Coalition is an independent 501(c)(3) nonprofit founded by volunteers in March 2020. Covid Act Now is our COVID-focused initiative to help people make informed decisions by providing timely and accurate data about COVID in the U.S.",
-  "learnLinks": [
-    {
-      "cta": "What can you do after you are fully vaccinated?",
-      "url": "/faq#what-can-you-do-after-fully-vaccinated"
-    },
-    {
-      "cta": "Can you transmit COVID after vaccination?",
-      "url": "/faq#can-you-transmit-covid-after-vaccination"
-    },
-    {
-      "url": "faq#can-you-get-covid-after-vaccination",
-      "cta": "Can you get COVID even after being vaccinated?"
-    }
-  ],
+  "learn": "Find simple definitions of COVID-related terms, deep dives into COVID science and treatments, and concise answers to pressing COVID questions.",
```

Comment:
Discussed with the team this morning whether adjusting the structure of the json here would not cause conflicts with Netlify. Consensus is it _should_ be fine.